2022.md

```diff
@@ -10,6 +10,7 @@
 * [Debug WSO2 products using IntelliJ](https://achinthaisuru444.medium.com/debug-wso2-products-using-intellij-f10b86c507f7) by [Achintha Isuru](https://achinthaisuru444.medium.com/)
 * [Getting started with GraphQL with Spring-boot](https://medium.com/@rajithacharith/getting-started-with-graphql-with-spring-boot-29dfac2f11d5) by [Charith Rajitha](https://medium.com/@rajithacharith)
 * [Writing Good Git Commit Messages](https://www.yasint.dev/good-git-commit-messages) by [Yasin](https://www.yasint.dev)
+* [Branding On Steroids](https://medium.com/@brionmario/branding-on-steroids-asgar-e90e19578dae?source=friends_link&sk=4161d6c200790955da33a5ea374a53a6) by [Brion Mario](https://medium.com/@brionmario)
```

Comment:
Let's update the blog count as well.

Reply:
Fixed with https://github.com/wso2/iam-blogs/pull/157/commits/b6b8138a67852ba216b58a0865a00b7dbe470ccd.